tools/lou_checkyaml.c

```diff
@@ -406,10 +484,18 @@ read_options(yaml_parser_t *parser, int len, int *xfail, translationModes *mode,
 			*mode = read_mode(parser);
 		} else if (!strcmp(option_name, "typeform")) {
 			yaml_event_delete(&event);
-			*typeform = read_typeforms(parser, len);
-		} else if (!strcmp(option_name, "cursorPos")) {
+			*typeform = read_typeforms(parser, wordLen);
+
+			// FIXME: Comparisons should be truly case insensitive, but apparently no standard function for that.
+		} else if (!strcmp(option_name, "inPos") || !strcmp(option_name, "inpos")) {
```

Comment:
Not sure if I like to API so relaxed, i.e. allowing multiple ways to specify the test ("inpos" and inPos"). Do we do that in other places too?

Reply:
No I don't think so. I agree that one way should be enough.